index.js

```diff
@@ -142,6 +142,7 @@ const featuresToIncludeTempUnprotectedExceptions = [
     'nonTracking3pCookies',
     'performanceMetrics',
     'phishingDetection',
+    'maliciousSiteProtection',
```

Comment:
I suspect we never wanted that here. This is why there are "exceptions"

Reply:
I thought we wanted exceptions in general, I just want to be able to define them manually though. If we remove it from here can we still do that in the `overrides/*.json`?

Reply:
nah just means temp allowlisted sites get an exception here. It's for website investigations where we don't know the cause.

We can still use overrides as you say :)